tsconfig.json

```diff
@@ -14,13 +14,8 @@
   "include": [
     "packages/core/src/util",
     "packages/core/src/tracing/*.js",
-    "packages/core/test/test_util/common_verifications.js",
     "packages/shared-metrics",
     "packages/collector/src"
```

Comment:
Question: What was the reason that only some public packages/exports are typed?

Reply:
I think the reason was purely that it is a lot of work to retroactively add types to existing code and Willian's plan was to do it iteratively.